admin/views/school/school-home.ejs

```diff
@@ -1,6 +1,16 @@
+
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-        <%- partial('partials/_name_and_sign_out') %>
+        <% if (user.role === 'HELPDESK') { %>
```

Comment:
Might be better to remove this magic string.  Something as important as role names might be kept in a frozen object literal.

Reply:
@jon-shipley replaced as per our discussion and your feedback.